src/api/cow/cow.spec.ts

```diff
@@ -0,0 +1,410 @@
+  signingScheme: SigningScheme.EIP712,
+}
+
+beforeEach(() => {
+  fetchMock.resetMocks()
+})
+
+afterEach(() => {
+  jest.restoreAllMocks()
+})
+
+test('Valid: Get an order ', async () => {
+  fetchMock.mockResponseOnce(JSON.stringify(ORDER_RESPONSE), { status: HTTP_STATUS_OK })
+  const order = await cowSdk.cowApi.getOrder(ORDER_RESPONSE.uid)
+  expect(fetchMock).toHaveBeenCalledTimes(1)
+  expect(fetchMock).toHaveBeenCalledWith(
+    `https://api.cow.fi/rinkeby/api/v1/orders/${ORDER_RESPONSE.uid}`,
+    FETCH_RESPONSE_PARAMETERS
+  )
+  expect(order?.uid).toEqual(ORDER_RESPONSE.uid)
+})
+
+test('Invalid: Get an order ', async () => {
+  fetchMock.mockResponseOnce(
+    JSON.stringify({
+      errorType: 'NotFound',
+      description: "You've passed an invalid URL",
+    }),
+    { status: HTTP_STATUS_NOT_FOUND }
+  )
+
+  try {
+    await cowSdk.cowApi.getOrder('notValidOrderId')
+  } catch (e: unknown) {
+    const error = e as OperatorError
+    expect(error.message).toEqual('Token pair selected has insufficient liquidity')
+    expect(fetchMock).toHaveBeenCalledTimes(1)
+    expect(fetchMock).toHaveBeenCalledWith(
+      'https://api.cow.fi/rinkeby/api/v1/orders/notValidOrderId',
+      FETCH_RESPONSE_PARAMETERS
+    )
+  }
+})
+
+test('Valid: Get last 5 orders for a given trader ', async () => {
+  const ORDERS_RESPONSE = Array(5).fill(ORDER_RESPONSE)
+  fetchMock.mockResponseOnce(JSON.stringify(ORDERS_RESPONSE), { status: HTTP_STATUS_OK })
+  const orders = await cowSdk.cowApi.getOrders({
+    owner: '0x00000000005ef87f8ca7014309ece7260bbcdaeb', // Trader
+    limit: 5,
+    offset: 0,
+  })
+  expect(fetchMock).toHaveBeenCalledTimes(1)
+  expect(fetchMock).toHaveBeenCalledWith(
+    'https://api.cow.fi/rinkeby/api/v1/account/0x00000000005ef87f8ca7014309ece7260bbcdaeb/orders/?limit=5',
+    FETCH_RESPONSE_PARAMETERS
+  )
+  expect(orders.length).toEqual(5)
+})
+
+test('Invalid: Get last 5 orders for an unexisting trader ', async () => {
+  fetchMock.mockResponseOnce(
+    JSON.stringify({
+      errorType: 'NotFound',
+      description: "You've passed an invalid URL",
+    }),
+    { status: HTTP_STATUS_NOT_FOUND }
+  )
+
+  try {
+    await cowSdk.cowApi.getOrders({
+      owner: 'invalidOwner',
+      limit: 5,
+      offset: 0,
+    })
+  } catch (e: unknown) {
+    const error = e as OperatorError
+    expect(error.message).toEqual('Token pair selected has insufficient liquidity')
+    expect(fetchMock).toHaveBeenCalledTimes(1)
+    expect(fetchMock).toHaveBeenCalledWith(
+      'https://api.cow.fi/rinkeby/api/v1/account/invalidOwner/orders/?limit=5',
+      FETCH_RESPONSE_PARAMETERS
+    )
+  }
+})
+
+test('Valid: Get last 5 trades for a given trader ', async () => {
+  const TRADES_RESPONSE = Array(5).fill(TRADE_RESPONSE)
+  fetchMock.mockResponseOnce(JSON.stringify(TRADES_RESPONSE), { status: HTTP_STATUS_OK })
+  const trades = await cowSdk.cowApi.getTrades({
+    owner: '0x00000000005ef87f8ca7014309ece7260bbcdaeb', // Trader
```

Comment:
This address is repeated couple of times in the tests. Could you use a const instead? Either a new one or reuse it from `TRADE_RESPONSE.owner`